committee-steering/governance/sig-governance.md

```diff
@@ -258,7 +258,7 @@ otherwise fulfill its Organizational Management responsibilities
 [The Release Team: Bug Triage, CI Signal, and more]: https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks
 [Production Readiness Reviewer]: /sig-architecture/production-readiness.md#becoming-a-prod-readiness-reviewer-or-approver
 [API Reviewer and Moderator]: /sig-architecture/api-review-process.md#expanding-the-reviewer-and-approver-pool
-[Marketing Council]: /communication/marketing-team/role-handbooks/council.md
+[Marketing Council]: /communication/contributor-comms/role-handbooks/council.md
```

Comment:
We should also make this "Contributor Comms" instead of "Marketing Council"

Reply:
But this file `council.md` doesn't exist, You mentioned earlier that it is because the marketing council is not there anymore, please correct me if I am wrong. So, what do you think I should do here?

Reply:
Can we replace it with [this role handbook](https://github.com/kubernetes/community/tree/master/communication/contributor-comms/role-handbooks)?

Reply:
@kaslin Can you please look into it

Reply:
Oh yes! Good point, I think what you did here works. Thanks for this!